src/colors/src/Colors.js

```diff
@@ -82,3 +82,100 @@ export const WHITE :string = '#ffffff';
  */
 
 export const OVERLAY_BG :string = rgba('#080019', 0.3);
+
+// V2 Color Families
+
+export const NEUTRAL = {
+  N00: '#F9F9FC',
```

Comment:
Doesn't stylelint complain about the uppercase?

https://github.com/stylelint/stylelint-config-standard/blob/master/index.js

Reply:
Oh, I guess stylelint won't complain here. Anyway, we use lowercase hex values.